flytekit/annotated/base_task.py

```diff
@@ -412,7 +413,7 @@ def get_registerable_entity(self) -> SdkTask:
 
     def get_fast_registerable_entity(self) -> SdkTask:
         entity = self.get_registerable_entity()
-        if entity.container is None:
+        if entity.container is None or entity.type == SdkTaskType.RAW_CONTAINER_TASK:
```

Comment:
Why are we using this constant? can we use from Annotated?

Reply:
good point! done